<launch>
 


    <node pkg="obj_detection" type="CameraStream" name="CameraStream" output="screen" >
	<param name="device" type="int" value="1" />
	<param name="img_path" type="string" value="/home/ste/Immagini/test_drone/"/>
	<param name="img_path_save" type="string" value="/home/ste/"/>
	<param name="real_time" type="bool" value="true"/>
	<param name="save_img" type="bool" value="true"/>
    </node>


        
</launch>
